test/core/xds/xds_http_filters_test.cc

```diff
@@ -896,6 +897,7 @@ TEST_P(XdsRbacFilterConfigTest, AllPrincipalTypes) {
 }
 
 TEST_P(XdsRbacFilterConfigTest, AuditLoggingOptions) {
```

Comment:
Please also add a test that if the env var is *not* set, we do not use the new fields even if they are set.

Reply:
Done. Thanks!